src/algebra/symmetrized.lean

```diff
@@ -0,0 +1,204 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.module.basic
+import tactic.abel
+
+/-!
+# Symmetrized algebra
+
+A commutative multiplication on a real or complex space can be constructed from any multiplication
+by "symmetrization" i.e
+```
+a∘b = 1/2(ab+ba).
+```
```

Comment:
Since this isn't lean code, it might make more sense to use LaTeX here (as `$$ a \circ b = \frac{1}{2}(ab + ba) $$` or whatever you think looks best)

Reply:
Okay, done. Thanks.